resources/views/alerting/logs/list.blade.php

```diff
@@ -2,22 +2,22 @@
 
 @include('includes.datatables')
 
-@section('title', trans('nav.overview.alerts.main') . ' ' .trans('nav.overview.alerts.log'))
+@section('title', trans('nav.overview.alerts.log'))
 
 @section('content-header')
     <h1>
-        {{ trans('nav.overview.alerts.main') }}
-        <small>{{ trans('nav.overview.alerts.log') }}</small>
```

Comment:
Don't like the small text after the large text?
I was thinking it would be:
Large: main task
Small: specific task context

But as you can see in the different files, I was trying out different things.